wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -502,13 +502,16 @@ body:not(.title-tagline-hidden) .site-branding-text {
 }
 
 .navigation-top {
-	position: relative;
 	z-index: 20;
 	background: #fff;
 	border: unset;
 	font-size: 14px;
 }
 
+.navigation-top .wrap {
+	display: flex;
```

Comment:
This works well for the desktop, but I noticed this on mobile:

<img width="557" alt="Screenshot 2023-05-03 at 8 22 17 AM" src="https://user-images.githubusercontent.com/484068/235962242-b6a91098-b0fd-4df1-b79c-d5db779a1e32.png">

I haven't come up with a good fix.

Reply:
Should also be fixed by the new wrapper.

Reply:
Good idea, looks great 👍🏻